### PR TITLE
Adding count rate in core

### PR DESCRIPTION
**Pull request by Vidushi-GitHub**
One property of the Einstein Probe is useful for various missions - net count rate.
However, it requires a consistent definition for other teams as well.
Example, some teams report count/s with background subtraction, others report count rates with background. 
To ensure consistency, it would be beneficial for the GCN team to standardize this property.

### CODE REVIEW

File: gcn/notices/core/Statistics.schema.json [hunk 1]
```diff
@@ -9,6 +9,14 @@
       "type": "number",
       "description": "False alarm rate: the rate of occurrence of non-astrophysical events that are of the same intensity or significance as the current event [Hz]"
     },
+    "net_count_rate": {
+      "type": "number",
+      "description": "Net count rate of the transient above the background [counts/s]"
+    },
+    "backgound_count_rate": {
+      "type": "number",
+      "description": "Mean count per second of the background for the transient [counts/s]"
```
Comment (Vidushi-GitHub) [inline gcn/notices/core/Statistics.schema.json:18]:
```suggestion
      "description": "Mean counts per second of the background for the transient [counts/s]"
```

File: gcn/notices/core/Statistics.schema.json [hunk 2]
```diff
@@ -9,6 +9,14 @@
       "type": "number",
       "description": "False alarm rate: the rate of occurrence of non-astrophysical events that are of the same intensity or significance as the current event [Hz]"
     },
+    "net_count_rate": {
+      "type": "number",
+      "description": "Net count rate of the transient above the background [counts/s]"
+    },
+    "backgound_count_rate": {
+      "type": "number",
+      "description": "Mean counts per second of the background for the transient [counts/s]"
```
Comment (lpsinger) [inline gcn/notices/core/Statistics.schema.json:18]:
```suggestion
      "description": "Count rate of the background for the transient [counts/s]"
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

**Review by lpsinger — APPROVED** [review]
Looks good, but I'd at least like @jracusin to also review this.

**Comment by Vidushi-GitHub** [comment]
@jracusin please check this PR.

File: gcn/notices/core/Statistics.schema.json [hunk 3]
```diff
@@ -9,6 +9,14 @@
       "type": "number",
       "description": "False alarm rate: the rate of occurrence of non-astrophysical events that are of the same intensity or significance as the current event [Hz]"
     },
+    "net_count_rate": {
+      "type": "number",
+      "description": "Net count rate of the transient above the background [counts/s]"
```
Comment (jracusin) [inline gcn/notices/core/Statistics.schema.json:14]: The description should specify that this is over the rate_duration and rate_energy_range.

File: gcn/notices/core/Statistics.schema.json [hunk 4]
```diff
@@ -9,6 +9,14 @@
...
+    },
+    "backgound_count_rate": {
+      "type": "number",
+      "description": "Count rate of the background for the transient [counts/s]"
```
Comment (jracusin) [inline gcn/notices/core/Statistics.schema.json:18]: Same comment about rate_duration and rate_energy_range as in the net_count_rate.

File: gcn/notices/core/Statistics.schema.json (hunk 3 above)
Comment (Vidushi-GitHub) [inline gcn/notices/core/Statistics.schema.json:14]:
```suggestion
      "description": "Net count rate of the transient above the background [counts/s] over rate_duration and rate_energy_range."
```

File: gcn/notices/core/Statistics.schema.json (hunk 4 above)
Comment (Vidushi-GitHub) [inline gcn/notices/core/Statistics.schema.json:18]:
```suggestion
      "description": "Count rate of the background for the transient [counts/s] over rate_duration and rate_energy_range."
```

File: gcn/notices/core/Statistics.schema.json [hunk 5]
```diff
@@ -9,6 +9,14 @@
       "type": "number",
       "description": "False alarm rate: the rate of occurrence of non-astrophysical events that are of the same intensity or significance as the current event [Hz]"
     },
+    "net_count_rate": {
+      "type": "number",
+      "description": "Net count rate of the transient above the background [counts/s] over rate_duration and rate_energy_range."
```
Comment (Vidushi-GitHub) [inline gcn/notices/core/Statistics.schema.json:14]:
```suggestion
      "description": "Net count rate of the transient above the background [counts/s] over rate_duration and rate_energy_range. Do specify rate_duration and rate_energy_range with net_count_rate."
```

File: gcn/notices/core/Statistics.schema.json [hunk 6]
```diff
@@ -9,6 +9,14 @@
       "type": "number",
       "description": "False alarm rate: the rate of occurrence of non-astrophysical events that are of the same intensity or significance as the current event [Hz]"
     },
+    "net_count_rate": {
+      "type": "number",
+      "description": "Net count rate of the transient above the background [counts/s] over rate_duration and rate_energy_range. Do specify rate_duration and rate_energy_range with net_count_rate property."
+    },
+    "backgound_count_rate": {
+      "type": "number",
+      "description": "Count rate of the background for the transient [counts/s] over rate_duration and rate_energy_range."
```
Comment (Vidushi-GitHub) [inline gcn/notices/core/Statistics.schema.json:18]:
```suggestion
      "description": "Count rate of the background for the transient [counts/s] over rate_energy_range."
```

Comment (Vidushi-GitHub) [reply gcn/notices/core/Statistics.schema.json:18]: @jracusin we can't use same rate_duration property for background. However we can add one for time range array for background.

File: gcn/notices/core/Statistics.schema.json [hunk 7]
```diff
@@ -9,6 +9,14 @@
...
+    },
```
Comment (Vidushi-GitHub) [inline gcn/notices/core/Statistics.schema.json:19]:
```suggestion
    },
    "backgound_rate_duration": {
       "type": "array",
       "description": "Multiple time Intervals for the background rate calculation [s]"
       },
```

File: gcn/notices/core/Statistics.schema.json [hunk 8]
```diff
@@ -9,6 +9,14 @@
       "type": "number",
       "description": "False alarm rate: the rate of occurrence of non-astrophysical events that are of the same intensity or significance as the current event [Hz]"
     },
+    "net_count_rate": {
+      "type": "number",
+      "description": "Net count rate of the transient above the background [counts/s] over rate_duration and rate_energy_range. Do specify rate_duration and rate_energy_range with net_count_rate property."
```
Comment (Vidushi-GitHub) [inline gcn/notices/core/Statistics.schema.json:14]:
```suggestion
      "description": "Net count rate of the transient above the background [counts/s] over rate duration and rate energy range. Do specify rate_duration and rate_energy_range with net_count_rate property."
```

File: gcn/notices/core/Statistics.schema.json (hunk 6 above)
Comment (jracusin) [reply gcn/notices/core/Statistics.schema.json]: The background rate comes from using multiple intervals, but it is used to estimate the background during the source interval.  I think this use is fine.

File: gcn/notices/core/Statistics.schema.json (hunk 7 above)
Comment (jracusin) [reply gcn/notices/core/Statistics.schema.json:23]: I do not think we need to include this.

**Comment by Vidushi-GitHub** [comment]
@jracusin background time intervals removed. Is everything else approved?

**Comment by Vidushi-GitHub** [comment]
Rebasing had some issues and we should notify these changes to Einstein Probe soon, so opened a new PR #136  and closing this.

**Pull request closed without merging**